include/internal/libfreenect2/async_packet_processor.h

```diff
@@ -119,6 +135,7 @@ class AsyncPacketProcessor : public PacketProcessor<PacketT>
       {
         // invoke process impl
         processor_->process(current_packet_);
+        releaseBuffer(current_packet_);
```

Comment:
ok, so basically every packetprocessor should call releaseBuffer at the end of process?